README.md

```diff
@@ -29,7 +29,10 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 Write your code here...
 
 ```ruby
-# code here
+def offer_rose (person)
+  puts "Would you take this rose, #{person}, in exchange for giving an old beggar woman shelter from the bitter cold?"
+
+end
 ```
 
```

Comment:
To complete your answer, call the method and pass in an argument: offer_rose("Young Prince")